dali/operators/reader/video_reader_op.cc

```diff
@@ -26,63 +26,83 @@ namespace dali {
 DALI_REGISTER_OPERATOR(VideoReader, VideoReader, GPU);
 
 DALI_SCHEMA(VideoReader)
-  .DocStr(R"code(
-Load and decode H264 video codec with FFmpeg and NVDECODE, NVIDIA GPU's hardware-accelerated video decoding.
-The video codecs can be contained in most of container file formats. FFmpeg is used to parse video containers.
-Returns a batch of sequences of `sequence_length` frames of shape [N, F, H, W, C] (N being the batch size and F the
-number of frames). Supports only constant frame rate videos.)code")
+  .DocStr(R"code(Loads and decodes the H264 video codec with FFmpeg and NVDECODE, which is
+the hardware-accelerated video decoding feature in the NVIDIA(R) graphical process unit (GPU).
+
+The video codecs can be in most of the container file formats. FFmpeg is used to parse video
+containers and returns a batch of sequences of sequence_length frames of the [N, F, H, W, C]
+shape, where N is the batch size, and F is the number of frames). This class only supports
+constant frame rate videos.)code")
   .NumInput(0)
   .OutputFn(detail::VideoReaderOutputFn)
   .AddOptionalArg("filenames",
-      R"code(File names of the video files to load.
-This option is mutually exclusive with `file_root` and `file_list`.)code",
+      R"code(Path to the file with a list of the file label pairs.
+
+This option is mutually exclusive with ``filenames`` and ``file_root``.)code",
```

Comment:
```suggestion
      R"code(File names of the video files to load.
      
This option is mutually exclusive with ``filenames`` and ``file_root``.)code",
```
Incorrect! This operator actually takes a list of file names, without extra indirection in the form of a flie list file.

Reply:
Done